stdlib/3.4/asyncio/tasks.pyi

```diff
@@ -19,10 +24,24 @@ def as_completed(fs: Sequence[_FutureT[_T]], *, loop: AbstractEventLoop = ...,
 def ensure_future(coro_or_future: _FutureT[_T],
                   *, loop: AbstractEventLoop = ...) -> Future[_T]: ...
 async = ensure_future
-# TODO: gather() should use variadic type vars instead of _TAny.
-_TAny = Any
-def gather(*coros_or_futures: _FutureT[_TAny],
-           loop: AbstractEventLoop = ..., return_exceptions: bool = False) -> Future[List[_TAny]]: ...
+@overload
+def gather(coro_or_future1: _FutureT[_T1],
+           loop: AbstractEventLoop = ..., return_exceptions: bool = False) -> Future[Tuple[_T1]]: ...
```

Comment:
The last overload could use `-> List[Any]` instead. I'm not sure what issues specifying the incorrect type hint here would imply for code that explicitly relies on that.

I'm opened to thoughts about how to better approach this or completely abandon this and just accept life with `cast`